charts/kubearchive/templates/sink/role.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ tpl .Values.sink.name . }}
```

Comment:
The bug found can be solved by suffixing this with `-configmaps` so the operator does not modify it.

Reply:
i went with `-watch` instead because I think it better describes what role is for